### ISSUE TITLE
Console: clicking on node link in pod view/list results in 404

### ISSUE THREAD

**Issue by cehoffman** [coreos/tectonic-installer#3059]
## What keywords did you search in tectonic-installer issues before filing this one?

console node

## Is this a BUG REPORT or FEATURE REQUEST?

BUG REPORT

### Tectonic version ([release](https://github.com/coreos/tectonic-installer/releases) or commit hash):
```
1.8.7-tectonic.2
```

### Platform (aws|azure|openstack|metal|vmware):

```
azure
```

### What happened?

```
Clicking on node from pod details or pod list results in a 404
```

### What you expected to happen?

```
Taken to node details
```

### How to reproduce it (as minimally and precisely as possible)?

```
Select a pod from running pods, then click on the node link for the node the pod is running on.
```

### Anything else we need to know?
```
This is an upgraded install from 1.8.4-tectonic.3 all the way back to 1.6.2
```

### References

**Comment by squat** [comment]
Cc @ggreer @kans

**Comment by kans** [comment]
The broken links have been fixed in master.  If you want, you can update the version of console your cluster is running by editing the `tectonic-console` in the `tectonic-sytem` namespace (`/k8s/ns/tectonic-system/deployments/tectonic-console/yaml`).  Change the image for the tectonic-console container to `quay.io/coreos/tectonic-console:v5.0.2`.

**Comment by cehoffman** [comment]
Thanks, I updated for us. If it is fixed upstream I guess there is no need for this to remain open.

**Comment by squat** [comment]
@kans we should PR that change to the track-1 branch so the upcoming track one releases can use this fixed version as well. Is this release compatible with k8s 1.8.x?

**Comment by ggreer** [comment]
This is fixed in console v5.0.4, which is compatible with k8s v1.8.

**Comment by squat** [comment]
Thanks, Geofffffffff
